src/App.jsx

```diff
@@ -155,13 +156,11 @@ const schemaShapes = {
 };
 
 function App() {
-  const years = [2019, 2020];
-  const [activeYear, setYear] = useLocalStorage("semestra-year", 2019);
+  const years = [2019, 2020, 2021];
+  const [activeYear, setYear] = useLocalStorage('semestra-year', 2019);
 
-  const [location, setLocation] = useLocalStorage(
-    "semestra-location",
-    "boston"
-  );
+  const locations = ['boston', 'uk', 'sweden', 'nyc', 'netherlands'];
```

Comment:
doesn't this make locations defined twice?

Reply:
yeah, no idea how that happened. Fixing :)